po/fr.po

```diff
@@ -156,15 +157,16 @@ msgstr "e.g. .thunderbird"
 #: src/models/persistent.js:62
 msgid "List of homedir-relative paths created in the sandbox"
 msgstr ""
-"Liste des chemins relatifs au répertoir personnel créé dans le sac de sable"
+"Liste des chemins relatif créés dans la sandox, avec comme point de "
```

Comment:
This probably won't fit in the section, and will get ellipsized?

Reply:
Indeed, it gets ellipsized. I reverted that change